### PR TITLE
Mad 1244/embed block

### PR DESCRIPTION
**Pull request by Heather0K**
possible solution for [MAD-1244](https://digirati.atlassian.net/browse/MAD-1244)

Screenshot 2023-02-07 at 16 29 48
Screenshot 2023-02-07 at 16 29 14

[MAD-1244]: https://digirati.atlassian.net/browse/MAD-1244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: services/madoc-ts/src/frontend/site/features/EmbedItem.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+`;
+export const EmbedItem: React.FC<{
+  link?: string;
+  height?: string;
+  width?: string;
+}> = ({ link, height, width }) => {
+  return (
+    <EmbedWrapper>
+      <iframe src={link} height={height} width={width} />
```
Comment (stephenwf) [inline services/madoc-ts/src/frontend/site/features/EmbedItem.tsx:17]: For security we should add some extra properties to limit what the iframe has access to.
- [`sandbox=""`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox)
- [`referrerpolicy="no-referrer"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-referrerpolicy)

There may be others we could add, but I think limiting what the frame can do would be enough for now. We could add options to the block later too.

Comment (Heather0K) [reply services/madoc-ts/src/frontend/site/features/EmbedItem.tsx]: `sandbox="" `dosent allow any scripts so most embeded widgets wont do much I added` sandbox="allow-scripts" `instead?

**Comment by Heather0K** [comment]
Screenshot 2023-02-07 at 17 54 35

File: services/madoc-ts/src/frontend/site/features/EmbedItem.tsx (hunk 1 above)
Comment (Heather0K) [reply services/madoc-ts/src/frontend/site/features/EmbedItem.tsx]: I can remove this block if theres no usecase for it?

**Review by stephenwf — APPROVED** [review]

File: services/madoc-ts/src/frontend/site/features/EmbedItem.tsx (hunk 1 above)
Comment (stephenwf) [reply services/madoc-ts/src/frontend/site/features/EmbedItem.tsx]: I think it still might be useful!

**Pull request merged**